spotifycli/spotifycli.py

```diff
@@ -49,6 +54,26 @@ def main():
         control_volume("-5%")
 
 
+def startShell():
```

Comment:
Whole script is formatted with the `snake_case` instead of `camelCase`. I'd like to keep it consistent.